src/transformers/models/opt/modeling_opt.py

```diff
@@ -63,6 +71,35 @@
 ]
 
 
+if is_torch_fx_available():
+
+    @torch.fx.wrap
+    def check_padding_in_attention_mask(attention_mask):
+        if 0 in attention_mask:
+            return attention_mask
+        return None
+
+else:
+
+    def check_padding_in_attention_mask(attention_mask):
+        if 0 in attention_mask:
+            return attention_mask
+        return None
```

Comment:
I will properly docstring this and move it inside `pytorch_utils` as it will be used by all models that will support FA2 and already supports fx tracing. I will take care of that in a separate PR.
```suggestion
# TODO: @younesbelkada move that in pytorch_utils and document it
if is_torch_fx_available():

    @torch.fx.wrap
    def check_padding_in_attention_mask(attention_mask):
        if 0 in attention_mask:
            return attention_mask
        return None

else:

    def check_padding_in_attention_mask(attention_mask):
        if 0 in attention_mask:
            return attention_mask
        return None
```

Reply:
done